### ISSUE TITLE
Tine shows only a blank page after update

### ISSUE THREAD

**Issue by mrHJS** [tine20/tine20#7491]
Hello everybody,

I downloaded the latest stable release 2023-11-1 and wanted to update the stable release I use from last year. So as every year, I simply copied the new files over the old ones and also copied the config file and the htaccess file.
However, when I now open the setup.php I only get a blank page, no source code. Same is happening when I try to open Tine in the browser.

I made a backup of the folder and when I revert to that, everything works again. 

I do not know what I am doing wrong, I have been doing updates for years this way and never had this problem.

Any advice is appreciated...

Edit:
Oh no, the old version is not working. It shows me the login screen but when I login I get the following error message:
Der Server kann Ihre Anfrage leider Aufgrund einer temporären Überlastung, Wartungsarbeiten oder Fehlkonfiguration nicht beantworten. Bitte versuchen Sie es später noch einmal oder kontaktieren Sie Ihren Administrator.

What happened? I am clueless...

I can login into setup.php - but cannot login into Tine.  Logger priority is set to 7, but no error is logged when I fail to login... 

Maybe it also has to do with the latest Synology Webserver update. When I change PHP to version 8.2 I suddenly get this error:
Fatal error: Uncaught Error: The DateTime object has not been correctly initialized by its constructor in [no active file]:0 Stack trace: #0 [internal function]: DateTime->__serialize() #1 {main} thrown in [no active file] on line 0 

But still not logged into my error.log file...

**Comment by mrHJS** [comment]
Ok...
so, I changed PHP to 8.2 and pointed to the new Tine 2.0 directory. Now it works again... ????
However, I seem to have downloaded a wrong version. The login page is telling:
tine ® Testversion.
Bitte kontaktieren Sie Metaways Infosystems GmbH um eine gültige Lizenz zu erwerben.

I thought this was the free stable version that is updated once per year. Last year I downloaded
it from here: https://packages.tine20.org/source/
This somehow did not work I ended up here https://github.com/tine-groupware/tine/releases where I downloaded the latest version.

I am very sorry if I made a mistake. Which version do I have to use?
